src/uct/sm/mm/mm_ep.c

```diff
@@ -58,10 +84,76 @@ ucs_status_t uct_mm_ep_put_bcopy(uct_ep_h tl_ep, uct_pack_callback_t pack_cb,
     return UCS_OK;
 }
 
-ucs_status_t uct_mm_ep_am_short(uct_ep_h ep, uint8_t id, uint64_t header,
+inline uint64_t uct_mm_ep_head_index_in_fifo(uct_mm_ep_t *ep)
+{
+	uct_mm_iface_t *iface = ucs_derived_of(ep->super.super.iface, uct_mm_iface_t);
+    return (ep->fifo_ctl->head & iface->fifo_mask);
+}
+
+uct_mm_fifo_element_t* uct_mm_ep_get_remote_elem(uct_mm_ep_t *ep, uint64_t head)
```

Comment:
Do we want to inline this function. I'm asking since it is used in critical path